cleverhans/evaluation.py

```diff
@@ -448,6 +482,49 @@ def __call__(self):
     return (x_batch, y_batch), (correct, max_probs)
 
 
+class _AttackFactory(object):
+  """
+  A factory for an expression that runs an adversarial attack
+  """
+
+  def __init__(self, model, attack=None, attack_params=None, pass_y=False):
```

Comment:
Can you add a comment for `pass_y`? It's possible to find out what it does from line 520 (pass the label placeholder to the attack) but it's not obvious when looking at init

Reply:
done